disambiguator.py

```diff
@@ -484,7 +484,7 @@ def create_features_labels(save_to_disk=False):
 
     # get features
     darrays = get_descriptor_arrays(tokens)
-    # labels = get_dummies(labels)
+    labels = get_dummies(labels)
 
     # put into grams (give context)
     darrays, labels = make_grams(
```

Comment:
I also made similar changes in the other branch... whoops -- i'll deal with the merge conflicts once @kennyhlam  approves the neuralnet branch